assets/controllers/markdown_toolbar_controller.js

```diff
@@ -0,0 +1,39 @@
+// SPDX-FileCopyrightText: 2023-2024 /kbin & Mbin contributors
+//
+// SPDX-License-Identifier: AGPL-3.0-only
+
+import { Controller } from '@hotwired/stimulus';
+
+/* stimulusFetch: 'lazy' */
+export default class extends Controller {
+    addSpoiler(event) {
+        event.preventDefault();
+
+        const input = document.getElementById(this.element.getAttribute('for'));
+        let spoilerBody = '_____';
```

Comment:
```suggestion
        let spoilerBody = 'spoiler body';
```
So users know that they should replace this text

Reply:
Yea ok, seems to be redundant and definitely confusing. I agree.